Unreal/CarlaUnreal/Plugins/Carla/Source/Carla/Settings/CarlaSettingsDelegate.cpp

```diff
@@ -232,12 +232,12 @@ void UCarlaSettingsDelegate::SetAllRoads(
     const float max_draw_distance,
     const TArray<FStaticMaterial> &road_pieces_materials) const
 {
-  if (!world || !IsValid(world) || world->IsPendingKill())
+  if (!world || !IsValid(world))
   {
     return;
   }
   AsyncTask(ENamedThreads::GameThread, [=]() {
-    if (!world || !IsValid(world) || world->IsPendingKill())
+    if (!world || !IsValid(world))
```

Comment:
```suggestion
    if (!IsValid(world))
```
IsValid() is already doing the nullcheck.